Find a functionally equivalent code:android.graphics.Matrix.postRotate:COMMENT
Method Modifier: public      
Comment:/**
 * Postconcats the matrix with the specified rotation.
 * M' = R(degrees) * M
 */

@@ -1,3 +1,4 @@
 {
-    return native_postRotate(native_instance, degrees);
+    native_postRotate(native_instance, degrees);
+    return true;
 }

